src/registry/local.rs

```diff
@@ -90,53 +90,45 @@ impl Registry for LocalRegistry {
 
 #[cfg(test)]
 mod tests {
-    use crate::manifest::{Dependency, PackageManifest};
-    use crate::package::{Package, PackageId, PackageType};
+    use crate::manifest::Dependency;
+    use crate::package::{Package, PackageId, PackageStore};
     use crate::registry::local::LocalRegistry;
     use crate::registry::Registry;
     use bytes::Bytes;
-    use semver::{Version, VersionReq};
+    use semver::VersionReq;
     use std::path::PathBuf;
     use std::str::FromStr;
     use std::{env, fs};
 
     #[tokio::test]
     async fn can_publish_and_fetch() {
-        let dir = env::temp_dir();
-        let registry = LocalRegistry::new(dir.clone());
-
-        let manifest = PackageManifest {
-            r#type: PackageType::Api,
-            name: PackageId::from_str("test-api").unwrap(),
-            version: Version {
-                major: 0,
-                minor: 1,
-                patch: 0,
-                pre: Default::default(),
-                build: Default::default(),
-            },
-            description: None,
+        let tmp_dir = env::temp_dir();
+        let registry = LocalRegistry::new(tmp_dir.clone());
+
+        // Create an (in-memory) release of the test proto package
+        let package_store = PackageStore {
+            base_dir: PathBuf::from("tests/data/test-api"),
         };
-        let package_bytes =
-            Bytes::from(include_bytes!("../../tests/data/test-api-0.1.0.tgz").to_vec());
+        let release = package_store.release().await.unwrap();
```

Comment:
this simplification is possible because we can now call `release` to create a release, rather than having to check in the pre-tarred tgz